### DISCUSSION TITLE
new component: carousel

### DISCUSSION THREAD

**Discussion by shoxton** [chakra-ui/chakra-ui#2624] [Ideas] [answered]
I dont know if you guys are already working on a carousel component, but it would be awesome having a chakra carousel built in component without needing to install additional packages.

**Comment by with-heart** [comment, marked as answer]
Agreed! I don't have a timeline, but I believe a carousel is something that @segunadebayo has expressed interest in supporting in the future.

**Comment by wryfi** [comment]
👍
